docs/source/files.rst

```diff
@@ -0,0 +1,43 @@
+
+
+===========================
+Files
+===========================
+
+.. automodule:: snowflake.snowpark.files
```

Comment:
Let's remove these two lines. It's causing an extra header `Attribute` to be generated at the end of [docs/api/snowflake.snowpark.files.SnowflakeFile.html#snowflake.snowpark.files.SnowflakeFile.html](url)

Reply:
When I remove these two lines we loose the title information, so I left them. Not sure what you are talking about with the attributes. When I ran it I don't see anything there. Unless it's something not obvious?

Reply:
![image](https://github.com/snowflakedb/snowpark-python/assets/77285980/577c98b5-a13f-4c77-8a9f-9afeb5ddf40e)
I was referring to the `Attributes` printed at the bottom. 
I see what you mean, since the documentation is very short, what if we just manually copy paste it in the `rst` file?

Reply:
Hrm I am a bit worried then it wouldn't get any updates which seems a bit worse. I added None to the attributes section. How is that? 
![Screenshot 2023-05-10 at 11 25 09 AM](https://github.com/snowflakedb/snowpark-python/assets/104799281/e63e64c5-8e90-4aa4-b4da-479492a3648d)